store/store_test.go

```diff
@@ -113,8 +113,8 @@ type testAuthContext struct {
 }
 
 func (ac *testAuthContext) Device() (*auth.DeviceState, error) {
-	freshDevice := *ac.device
-	return &freshDevice, nil
+	freshDevice := ac.device
```

Comment:
bit worried that we don't do a copy anymore, also  the real AuthoContext.Device never returns nil for device I think, we should probably follow that